src/main/java/cpsc/dlsproject/ast/ASTHelpers.java

```diff
@@ -0,0 +1,32 @@
+package cpsc.dlsproject.ast;
+
+import cpsc.dlsproject.tools.Tokenizer;
+
+public class ASTHelpers {
+
+    public static boolean CheckForRequestType() {
+        Tokenizer tokenizer = Tokenizer.getTokenizer();
+        return tokenizer.checkToken("GET") || tokenizer.checkToken("POST") || tokenizer.checkToken("DELETE") || tokenizer.checkToken("PUT");
+    }
+
+    public static boolean CheckForCond() {
+        Tokenizer tokenizer = Tokenizer.getTokenizer();
+        return tokenizer.checkToken("IF") || tokenizer.checkToken("ELSE") || tokenizer.checkToken("THEN");
+    }
+
+    public static boolean CheckForIO() {
+        Tokenizer tokenizer = Tokenizer.getTokenizer();
+        return tokenizer.checkToken("READFILE") || tokenizer.checkToken("WRITEFILE");
+    }
+
+    public static boolean CheckForLogical() {
+        Tokenizer tokenizer = Tokenizer.getTokenizer();
+        return  tokenizer.checkToken("<") || tokenizer.checkToken(">") || tokenizer.checkToken("==") ||
```

Comment:
Could make the code more concise by having check token to receive array of symbols and check if it matches one of them.

```
tokenizer.checkTokenAny(["<", ">", "==", "!=", "<=", ">="]);
```